source/_components/sensor.sense.markdown

```diff
@@ -31,6 +31,9 @@ sensor:
     - active_production
     - daily_usage
     - daily_production
+  devices:
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.